es-common-extensions/package.json

```diff
@@ -4,14 +4,14 @@
   "private": true,
   "scripts": {
     "start": "vite dev",
-    "build": "vite build",
+    "build": "vite build && mkdir -p dist/dist && cp -rf dist/*.js dist/dist/",
```

Comment:
Vite has two modes: dev and preview. In dev mode, files are served directly from the source, whereas preview mode serves files from the dist directory, which contains the production build. In this case, it was necessary to move some of the extensions into another 'dist' folder in order for the paths to work out in preview mode.